src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -44,16 +44,16 @@ public class EditCommand extends Command {
             + "[" + PREFIX_NAME + "NAME] "
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
-            + "[" + PREFIX_ADDRESS + "ADDRESS] "
-            + "[" + PREFIX_TAG + "TAG]... "
-            + "[" + PREFIX_COMMENT + "COMMENT]...\n"
+            + "[" + PREFIX_TELEGRAM_HANDLE + "TELEGRAM HANDLE] "
+            + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_PHONE + "91234567 "
             + PREFIX_EMAIL + "johndoe@example.com";
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists "
+            + "in the address book. Either the name or telegram handle already exist";
```

Comment:
Minor nit - the error message should more succinctly just write "This person or telegram handle already exists."

Reply:
changed to this person and telegram handle